web/src/routes/signIn/SignIn.svelte

```diff
@@ -52,8 +52,11 @@
 	 * Handles the submit event of the form.
 	 * @param e Submit event.
 	 */
-	async function handleSubmit(e: SubmitEvent) {
-		const formData = new FormData(e.currentTarget as HTMLFormElement);
+	async function handleSubmit(
+		e: Event & { currentTarget: EventTarget & HTMLFormElement },
```

Comment:
Do we need to perform this modification for the employees as well?
- https://github.com/Goncalo-Marques/ecomap/pull/205

Reply:
Yes. Ideally, every form event submission should now have this type for the event.